lib/logstash/inputs/gelf.rb

```diff
@@ -68,36 +76,135 @@ def initialize(params)
   public
   def register
     require 'gelfd'
+    @tcp = nil
```

Comment:
Is tcp and udp (actual named vars used?)

Or are these ment to be use_tcp and use_udp?